src/katgpucbf/fgpu/main.py

```diff
@@ -226,13 +233,14 @@ def field_callback(kws: NarrowbandOutputDict, key: str, data: str) -> None:
         kws = {
             "taps": DEFAULT_TAPS,
             "w_cutoff": DEFAULT_W_CUTOFF,
+            "jones_per_batch": DEFAULT_JONES_PER_BATCH,
             "weight_pass": DEFAULT_WEIGHT_PASS,
             "ddc_taps": DEFAULT_DDC_TAPS_RATIO * kws["decimation"],
-            **kws,  # type: ignore[misc]
```

Comment:
No need for the `# type: ignore` anymore?

Reply:
No, mypy fixed the bug.